### PR TITLE
Update detail map documentation

### PR DESCRIPTION
**Pull request by adrien-de-tocqueville**
## Purpose of this PR

Add information about the detail map and import settings.
Images to show the impact of detail albedo and smoothness on a lit shader

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/Mask-Map-and-Detail-Map.md [hunk 1]
```diff
@@ -44,3 +46,19 @@ The detail map contains two grayscale Textures and one two-component Texture, wh
 The following example image demonstrates the individual components of a full detail map.
 
 ![](Images/MaskMapAndDetailMap3.png)
+
+### Desaturated albedo (red channel)
+
+The red channel represents the albedo variation. It can make the underlying material's albedo gradually darken down to black when going from `0.5` to `0` or brighten up to white when going from `0.5` to `1`. A value of `0.5` is neutral, which means the albedo won't be modified by detail map.
+
+The image below shows the impact of the detail albedo on the final color. As you can see, color interpolation is done in sRGB space.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Mask-Map-and-Detail-Map.md:54]: HDRP calculates color interpolation in sRGB space.

File: com.unity.render-pipelines.high-definition/Documentation~/Mask-Map-and-Detail-Map.md [hunk 2]
```diff
@@ -44,3 +46,19 @@ The detail map contains two grayscale Textures and one two-component Texture, wh
 The following example image demonstrates the individual components of a full detail map.
 
 ![](Images/MaskMapAndDetailMap3.png)
+
+### Desaturated albedo (red channel)
+
+The red channel represents the albedo variation. It can make the underlying material's albedo gradually darken down to black when going from `0.5` to `0` or brighten up to white when going from `0.5` to `1`. A value of `0.5` is neutral, which means the albedo won't be modified by detail map.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Mask-Map-and-Detail-Map.md:52]: A value of `0.5` is neutral, which means the detail map does not modify the albedo.

File: com.unity.render-pipelines.high-definition/Documentation~/Mask-Map-and-Detail-Map.md [hunk 3]
```diff
@@ -32,7 +32,9 @@ The following example image demonstrates the individual components of a full mas
 
 ## Detail map
 
-The detail map contains two grayscale Textures and one two-component Texture, which is the Material's normal map.
+The detail map allow you to overlay a second set of textures on top of the base surface information. Typically, the detail map would be scaled several times across the object’s surface to add small details to a material.
+The detail map contains two grayscale Textures and one two-component Texture, which is the Material's detail normal map.
+When importing the texture, make sure the `sRGB` checkbox is not ticked in the Import Settings.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Mask-Map-and-Detail-Map.md:37]: When you import the detail map texture, make sure to disable the **sRGB** checkbox in the Import Settings.

File: com.unity.render-pipelines.high-definition/Documentation~/Mask-Map-and-Detail-Map.md [hunk 4]
```diff
@@ -44,3 +46,19 @@ The detail map contains two grayscale Textures and one two-component Texture, wh
...
+
+The red channel represents the albedo variation. It can make the underlying material's albedo gradually darken down to black when going from `0.5` to `0` or brighten up to white when going from `0.5` to `1`. A value of `0.5` is neutral, which means the albedo won't be modified by detail map.
+
+The image below shows the impact of the detail albedo on the final color. As you can see, color interpolation is done in sRGB space.
+
+![](Images/DetailMap-red.png)
+
+### Smoothness (blue channel)
+
+The blue channel represents the smoothness variation and is computed the same way as the albedo variation. The underlying material's smoothness will be gradually decreased if the detail smoothness is below `0.5` or increased if it is above `0.5`. A value of `0.5` is neutral, which means the smoothness won't be modified by detail map.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Mask-Map-and-Detail-Map.md:60]: The blue channel represents the smoothness variation and HDRP computes it the same way as the albedo variation. The underlying material's smoothness gradually decreases if the detail smoothness is below `0.5` or increases if it is above `0.5`. A value of `0.5` is neutral, which means the detail map does not modify the smoothness.

**Comment by adrien-de-tocqueville** [comment]
Something like that ? (with text)
base

**Review by sebastienlagarde — APPROVED** [review]

**Pull request merged**